CHANGELOG.md

```diff
@@ -1,3 +1,114 @@
+### citus v9.5.11 (April 20, 2023) ###
+
+* Fixes a bug that could break `DROP SCHEMA/EXTENSON` commands when there is a
+  columnar table (#5458)
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker (#5679)
+
+* Fixes memory leak issue with query results that returns single row (#6724)
+
+* Prevents alter table functions from dropping extensions (#5974)
+
+### citus v10.0.7 (April 20, 2023) ###
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker (#5679)
+
+* Fixes columnar freezing/wraparound bug (#5962)
+
+* Fixes memory leak issue with query results that returns single row (#6724)
+
+* Prevents alter table functions from dropping extensions (#5974)
+
+### citus v10.1.5 (April 20, 2023) ###
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker (#5679)
+
+* Fixes columnar freezing/wraparound bug (#5962)
+
+* Fixes memory leak issue with query results that returns single row (#6724)
+
+* Prevents alter table functions from dropping extensions (#5974)
+
+### citus v10.2.9 (April 20, 2023) ###
+
+* Correctly reports shard size in `citus_shards` view (#6748)
+
+* Fixes a bug in `ALTER EXTENSION citus UPDATE` (#6383)
+
+* Fixes a bug that prevents retaining columnar table options after a
+  table-rewrite (#6337)
+
+* Fixes memory leak issue with query results that returns single row (#6724)
+
+* Raises memory limits in columnar from 256MB to 1GB for reads and writes
+  (#6419)
+
+### citus v11.0.8 (April 20, 2023) ###
```

Comment:
I have problems coming up with changelog items for 3 commits on release-11.0 branch.

1. https://github.com/citusdata/citus/commit/1f03f136659554619b2bd6f43767cc909d9088f0 did not use `-x` when cherry picking. I have trouble finding the PR number and changelog entry if there is one. @agedemenli
2. https://github.com/citusdata/citus/pull/6624 is cherry-picked to release-11.1 branch. This PR is shipped with 11.2 release but the changelog for that version do not seem to contain a similar entry that is supplied in the PR description. @onurctirtir

Reply:
For 1): This is the originating issue https://github.com/citusdata/citus/pull/6603, with DESCRIPTION: "Fix the modifier for subscription and role creation"